{"name": "Ошибка 409 Сonflict при повторной регистрации курьера с валидными \"login\", \"password\" и \"firstName\".", "status": "passed", "description": "Генерируем данные курьера фикстурой courier_payload, регистрируем его в системе.Отправлям повторный POST запрос на ручку /api/v1/courier для регистрации с теми же данными. Проверяем, что ответ 409 Сonflict и тело ответа {\"message\": \"Этот логин уже используется\"}. Созданный курьер удаляется из системы фикстурой courier_payload.", "steps": [{"name": "POST request.", "status": "passed", "parameters": [{"name": "path", "value": "'/api/v1/courier'"}, {"name": "payload", "value": "{'login': 'lgxdjapkbi', 'password': 'etwlbarzzr', 'firstName': 'wtbhkcwttq'}"}], "start": 1739134607582, "stop": 1739134607625}, {"name": "POST request.", "status": "passed", "parameters": [{"name": "path", "value": "'/api/v1/courier'"}, {"name": "payload", "value": "{'login': 'lgxdjapkbi', 'password': 'etwlbarzzr', 'firstName': 'wtbhkcwttq'}"}], "start": 1739134607626, "stop": 1739134607670}], "start": 1739134607582, "stop": 1739134607671, "uuid": "08450611-380d-4d67-afcb-a4a63a5563ba", "historyId": "398c7ae9bc0026a5edfc88dde6c0d635", "testCaseId": "398c7ae9bc0026a5edfc88dde6c0d635", "fullName": "tests.test_create_courier.TestCreateCourier#test_create_courier_same_login_details_shows_error_409", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_create_courier"}, {"name": "subSuite", "value": "TestCreateCourier"}, {"name": "host", "value": "k-julls"}, {"name": "thread", "value": "10016-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_create_courier"}]}